pkg/util/token/bootstrap_token.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package token
```

Comment:
i think these utils should live in https://github.com/kubernetes/cluster-bootstrap
https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/cluster-bootstrap/token/util

Reply:
Yeah, they seem more relevant there.

Reply:
That was my first thought. But that means we need to import `"k8s.io/api/core/v1"` in package `k8s.io/cluster-bootstrap/`.  For potential import cycle issues, do we really want to do that?

Reply:
cluster-bootstrap already includes `k8s.io/api/core/v1` but only under the api package:
https://github.com/kubernetes/cluster-bootstrap/blob/master/token/api/types.go#L20
i guess the import cycle concerns are valid.

Reply:
@neolit123 @smarterclayton Moved out to `k8s.io/cluster-bootstrap`. PTAL. Thanks